dev-tools/packaging/templates/linux/postinstall.sh.tmpl

```diff
@@ -16,8 +16,9 @@ if test -L "$symlink"; then
 fi
 
 commit_hash="{{ commit_short }}"
+version_dir="{{agent_package_version}}{{snapshot_suffix}}"
```

Comment:
nit: name suggests it's the name of the whole dir. what's the reason of this separation.can we either include it in new_agent_dir or exclude whole dir into this one?

Reply:
We can certainly nest this in the new_agent_dir variable directly, I thought that for debugging the script this would be easier than a big string concatenation done all at once (similar to what is done for the commit_hash